index.html

```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <meta name="robots" content="noindex, nofollow">
```

Comment:
This line needs to be in `2016/index.html`, not `index.html`. We want search engines to ignore the old site, not the new one :smiley: